src/Microsoft.Azure.IIoT.OpcUa.Testing/cli/Program.cs

```diff
@@ -671,7 +693,14 @@ private class ModelWriter : IEventEmitter {
             var storage = new ZipArchiveStorage();
 
             var fileName = "tmp.zip";
-            using (var client = new ClientServices(logger.Logger))
+            Environment.SetEnvironmentVariable("AutoAccept", "true");
```

Comment:
While this works I would either a) make a helper method (since lots of repetition), or b) (preferred) just create a quick mock class that implements IClientServicesConfig and returns true for AutoAccept.